tango/common/file_lock.py

```diff
@@ -24,10 +24,12 @@ def __init__(self, lock_file: PathOrStr, timeout=-1, read_only_ok: bool = False)
         super().__init__(str(lock_file), timeout=timeout)
         self._read_only_ok = read_only_ok
 
-    @overrides
-    def acquire(self, timeout=None, poll_intervall=0.05) -> AcquireReturnProxy:
+    # TODO (epwalsh): remove the `check_signature=False` when filelock removes the deprecated
+    # misspelled `poll_intervall` parameter.
```

Comment:
I'm a little confused why this is necessary now but I trust you know what you're doing.